MapboxCoreNavigation.podspec

```diff
@@ -41,7 +41,7 @@ Pod::Spec.new do |s|
   s.module_name = "MapboxCoreNavigation"
 
   s.dependency "MapboxNavigationNative", "~> 101.0"
-  s.dependency "MapboxDirections-pre", "2.5.0-rc.1"
+  s.dependency "MapboxDirections", "~> 2.5.0"
```

Comment:
For SPM we have a range of `2.5.0..<3.0.0` while for CocoaPods and Carthage it's `2.5.0..<2.6.0`. I would say we should have `~> 2.5` here.

Reply:
Thanks for the suggestion, I'll update this part.